docker/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ $# -ne 1 ]]; then
+  echo "Expected 1 parameter, but got $#."
+
+  cat <<USAGE_DOC
+Usage: build.sh <VERSION>
+
+Builds the gitkv binary for the given version and tags the container as 'gitkv:<VERSION>'.
+
+Parameters:
+    VERSION: The version to build, in the format 'vX.Y.Z' — note the 'v' prefix is needed.
+USAGE_DOC
+  exit 1
+fi
+
+VERSION=$1
+
+echo "Building gitkv:$VERSION"
+
+docker build -t gitkv:$VERSION --network=host --build-arg version=$VERSION .
```

Comment:
I _think_ we shouldn't need the `--network=host` flag here — that's normally an argument for `docker run`.